src/main/java/PTBac2.java

```diff
@@ -0,0 +1,15 @@
+public class PTBac2 {
+    public String giaiphuongtrinh(double a, double b, double c){
+        String s="";
+        double delta = b*b -4*a*c;
+
+        if (delta < 0){
+            s="Phương trình vô nghiệm";
+        }
+        else{
+            s="x1= " + ((-1*b+Math.sqrt(delta))/2/a) + ", x2= " + ((-1*b-Math.sqrt(delta))/2/a);
```

Comment:
Dòng quá dài, phải khai báo 2 biến x1 x2 để nó ngắn hơn. Và chưa check nếu có 1 nghiệm

Reply:
Dạ ý tưởng của em là vd có 1 nghiệm là 2 thì in ra "x1=2, x2=2" ấy a

Reply:
Dạ em sửa rồi ạ